main.go

```diff
@@ -423,6 +447,18 @@ func run(c *cli.Context) error {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
+	// Apply Kubernetes secrets manifests
+
+	if len(manifestsSecret) > 0 {
```

Comment:
I think we can improve the logging and tell the user when there is no Secret manifest configured.

Reply:
[This line](https://github.com/NYTimes/drone-gke/pull/66/files#diff-7ddfb3e035b42cd70649cc33393fe32cR333) will print when there's no no Secret manifest. or do we need more / different logging?

Reply:
You are right, that looks good to me.